### ISSUE TITLE
Check if all working groups are listed

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amp.dev#3384]
At least UI & Accessibility is missing.

https://amp.dev/community/working-groups/

**Comment by sebastianbenz** [comment]
//cc @nainar

**Comment by matthiasrohmer** [comment]
Apparently the ampproject/wg-ui-and-a11y working group never merged the related PR: https://github.com/ampproject/wg-ui-and-a11y/pull/28 😁

**Comment by DerekNonGeneric** [comment]
Are you aware that https://amp.dev/community/working-groups/ is currently a 404?

![wg-404](https://user-images.githubusercontent.com/17770407/72560791-c5dac780-3875-11ea-804c-a56d3381a847.png)

**Comment by matthiasrohmer** [comment]
Thanks, @DerekNonGeneric! It's not explicitly linked anywhere this way, is it? Anyway the URL would be nice to have.

**Comment by DerekNonGeneric** [comment]
> It's not explicitly linked anywhere this way, is it?

I'm not sure, but I did not arrive at that page via a link within the site, so it might not be too bad.

This might be another page that would be a good candidate for a section overview page (in addition to Support as is made evident in https://github.com/ampproject/amp.dev/issues/3282).

**Comment by robinvanopstal** [comment]
I had a closer look at that and to me, it seems that the following working groups are still missing: 

- wg-approvers
- wg-bento  
- wg-codeofconduct
- wg-foundation-onboarding
- wg-security-privacy

**Comment by robinvanopstal** [comment]
Missing PRs are now created and ready for review: 

- wg-approvers: https://github.com/ampproject/wg-approvers/pull/1
- wg-bento: https://github.com/ampproject/wg-bento/pull/3
- wg-codeofconduct: https://github.com/ampproject/wg-codeofconduct/pull/5
- wg-foundation-onboarding: https://github.com/ampproject/wg-foundation-onboarding/pull/24
- wg-security-privacy: https://github.com/ampproject/wg-security-privacy/pull/1

**Comment by matthiasrohmer** [comment]
Woop, woop. All of them are merged and almost all of them made it to amp.dev. An inline edit to the https://github.com/ampproject/wg-security-privacy/pull/1 one caused a YAML syntax error. @robinvanopstal can you please issue a fix PR?

```
Failed loading ampproject/wg-security-privacy/METADATA.yaml YAMLException: bad indentation of a mapping entry at line 14, column 7:
          "#wg"
          ^
    at generateError (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:167:10)
    at throwError (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:173:9)
    at readBlockMapping (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1107:7)
    at composeNode (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1359:12)
    at readBlockSequence (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:955:5)
    at composeNode (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1358:12)
    at readBlockMapping (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1089:11)
    at composeNode (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1359:12)
    at readDocument (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1519:3)
    at loadDocuments (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1575:5)
    at load (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1596:19)
    at Object.safeLoad (/home/travis/build/ampproject/amp.dev/node_modules/js-yaml/lib/js-yaml/loader.js:1618:10)
    at Object.importWorkingGroups (/home/travis/build/ampproject/amp.dev/gulpfile.js/import.js:55:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Promise.all (index 3) {
  name: 'YAMLException',
  reason: 'bad indentation of a mapping entry',
  mark: Mark {
    name: null,
    buffer: 'title: Security & Privacy\n' +
      'description: The wg-security-privacy Working Group is responsible for ensuring AMP maintains high standards with respect to security and privacy, including documenting security/privacy best practices and facilitating security/privacy reviews for AMP code and features.\n' +
      'facilitator:\n' +
      '  login: honeybadgerdontcare\n' +
      '  name: David Sedano\n' +
      'members:\n' +
      '  - login: molnarg\n' +
      '    name: Gábor Molnár\n' +
      'communication:\n' +
      '  - channel: slack\n' +
      '    name: Slack\n' +
      '    content:\n' +
      '      - item: "Security & Privacy Working Group members will use `#wg-security-privacy` channel on AMP\'s Slack ([signup](https://docs.google.com/forms/d/e/1FAIpQLSd83J2IZA6cdR6jPwABGsJE8YL4pkypAbKMGgUZZriU7Qu6Tg/viewform?fbzx=4406980310789882877)) for real-time discussion. The channel is open to anyone, regardless of membership in Secuirty & Privacy working group."\n' +
      '      "#wg"\n' +
      '\u0000',
    position: 846,
    line: 13,
    column: 6
  },
  message: 'bad indentation of a mapping entry at line 14, column 7:\n' +
    '          "#wg"\n' +
    '          ^'
}
